starter-code/pizza.js

```diff
@@ -1 +1,58 @@
 // Write your Pizza Builder JavaScript in this file.
+
+$(".btn-pepperonni").click(function(e){
+  $(this).toggleClass('active');
+  $('.pep').toggle();
+  $("li:contains('pepperonni')").toggle();
+  calcPrice();
+});
+
+$(".btn-mushrooms").click(function(e){
+  $(this).toggleClass('active');
+  $('.mushroom').toggle();
+  $("li:contains('mushrooms')").toggle();
+  calcPrice();
+});
+
+$(".btn-green-peppers").click(function(e){
+  $(this).toggleClass('active');
+  $('.green-pepper').toggle();
+  $("li:contains('green peppers')").toggle();
+  calcPrice();
+});
+
+$(".btn-sauce").click(function(e){
+  $(this).toggleClass('active');
+  $('.sauce').toggle();
+  $("li:contains('white sauce')").toggle();
+  calcPrice();
+});
+
+$(".btn-crust").click(function(e){
+  $(this).toggleClass('active');
+//  $('.crust').toggle();
```

Comment:
The price gets adjusted but the pizza looks the same.  You took out the $('.crust').toggle(); probably because you didn't want to take away the crust completely, but the crust has 2 classes `crust` and `crust-gluten-free` if you take away the `crust-gluten-free` class, it will look like a regular crust.